### PR TITLE
Login and modular fe

### PR DESCRIPTION
**Pull request by abeersaqib**
1. FE is converted to modular parts
2. Vite redirect added for Login

Code deployed at: https://staging.webaverse.com

### CODE REVIEW

File: index.mjs [hunk 1]
```diff
@@ -53,6 +53,9 @@ function makeId(length) {
         req.originalUrl = u;
         next();
       }
+    } else if (/^\/login/.test(o.pathname)) {
```
Comment (avaer) [inline index.mjs:56]: `/^\/login$/`?

**Review by avaer — CHANGES_REQUESTED** [review]
Can this be two PR's?

This has a very high likelihood of being reverted when a regression is discovered so let's make that clean to do so.

File: index.mjs (hunk 1 above)
Comment (abeersaqib) [reply index.mjs:56]: $ will match with the end of the string. While our express don't end at login it has id & code as well.

**Comment by abeersaqib** [comment]
> Can this be two PR's?
> 
> This has a very high likelihood of being reverted when a regression is discovered so let's make that clean to do so.

Sure

**Pull request closed without merging**